### DISCUSSION TITLE
Hardware requirements

### DISCUSSION THREAD

**Discussion by dwiehoff** [mediar-ai/screenpipe#339] [Q&A]
Wasn't able to find anything: I'm on a 2018 MBP, would I be able to run this?

**Comment by louis030195** [comment]
@dwiehoff sorry did not see this

screenpipe uses between 600mb to 4gb of ram and 10-30% CPU (on mac book pro m3) depending on your settings and approximately 30gb of storage per month

is that intel mac?

you can get the app here https://screenpi.pe/onboarding

if unhappy just let me know will refund you

**Comment by Mertyandimata** [comment]
Its amazing !! but I scared run on m1 mac air  :D

**Reply by louis030195** [reply]
@Mertyandimata you can still lower settings although i run screenpipe on a $200 windows laptop too

**Comment by louis030195** [comment]
hey @dwiehoff @Mertyandimata

closing this as part of our cleanup - screenpipe v2.2.0 is now out with major improvements!

check it out: https://x.com/louis030195/status/2014403576475615312

if this is still relevant or you have questions, feel free to reopen this discussion. thanks for being part of the community!
